book/chapter-08-tupperware.md

```diff
@@ -1,8 +1,21 @@
 # Chapter 08: Tupperware
 
+
+
+{% hint style="info" %}
+ The book often uses the "Point" style in its examples.  
+
+```text
+Container.of('bombs').map(append(' away')).map(prop('length')); 
+// Container(10)
```

Comment:
@cdimitroulas not sure if we can really provide a meaningful translation for `Container` so I just added a note about the use of point style